.github/workflows/cd-workflow.yml

```diff
@@ -0,0 +1,25 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Why was main added here? I think main is way behind the current progress, and we can just merge it in master to get rid of it

Reply:
Sorry, I had the impression that we were eventually going to switch to `main` as the main branch. I can remove it if that is not the case.